flow/connectors/core.go

```diff
@@ -38,6 +38,13 @@ type CDCPullConnector interface {
 	EnsurePullability(ctx context.Context, req *protos.EnsurePullabilityBatchInput) (
 		*protos.EnsurePullabilityBatchOutput, error)
 
+	// For InitialSnapshotOnly correctness without replication slot
+	// `any` is for returning transaction if necessary
+	ExportSnapshot(context.Context) (string, any, error)
```

Comment:
can we leave these methods out of the interface and use them directly from `PostgresConnector`?

Reply:
this and a lot of other methods are very PG specific, no point in making them an interface

Reply:
Activities should be connector agnostic. I can move this to a separate connector interface tho